src/twisted/conch/scripts/ckeygen.py

```diff
@@ -220,7 +245,7 @@ def _saveKey(key, options):
     @param options:
     @type options: L{dict}
     """
-    keyTypeName = key.type().lower()
+    keyTypeName = 'ecdsa' if key.type() == 'EC' else key.type().lower()
```

Comment:
This seems like it would accumulate more special cases with more key types. Should we give up and just use an explicit dictionary?